src/sentry/api/endpoints/organization_integrations.py

```diff
@@ -5,22 +5,24 @@
 )
 from sentry.api.paginator import OffsetPaginator
 from sentry.api.serializers import serialize
-from sentry.models import Integration
+from sentry.models import OrganizationIntegration
 
 
 class OrganizationIntegrationsEndpoint(OrganizationEndpoint):
     permission_classes = (OrganizationIntegrationsPermission, )
 
     def get(self, request, organization):
-        integrations = Integration.objects.filter(organizations=organization)
+        integrations = OrganizationIntegration.objects.filter(organization=organization)
 
```

Comment:
and this can go to maybe `def get_queryset()` ?  to make it look very django CBV-ic ?

Reply:
We don't use Django CBV here, so I think it's worse to try to mimic that API when it doesn't exist here.

Reply:
okey. I see its inheriting from `from rest_framework.views import APIView`, but again - the point is if you want to repeat the fetch in every other http method that will get overrident at some point. but I see you `think its worse..`, so probably I will leave it to your judgement  then.